products/yarn.md

```diff
@@ -36,8 +36,8 @@ releases:
 
 -   releaseCycle: "1"
     releaseDate: 2017-09-05
-    eol: false
-    latestReleaseDate: 2022-06-08
+    eol: 2022-05-10
+    latestReleaseDate: 2022-05-10
```

Comment:
Latest release date is automatically updated by our automation based on https://www.npmjs.com/package/yarn/v/1.22.19 (which is marked as released on 2022-06-08). Even if set manually it will be overwritten on the next auto-update (not an issue IMO given the relatively small difference).

Reply:
we can use the auto generated date. I have reverted this line.